airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -232,6 +232,11 @@ def make_pod(self, namespace, worker_uuid, pod_id, dag_id, task_id, execution_da
         affinity = kube_executor_config.affinity or self.kube_config.kube_affinity
         tolerations = kube_executor_config.tolerations or self.kube_config.kube_tolerations
 
+        security_context = {
+            'run_as_user': self.kube_config.worker_run_as_user,
+            'fs_group': self.kube_config.worker_fs_group,
```

Comment:
Is it okay to have (for example) `fs_group: ""` which is I think what we would get if one of these options is provided in the config file but not the other.

Reply:
Yes, it would not cause any issues as if a field is empty the api-server ignores it.

Nevertheless I will refactor this block because this made me realize that the condition on line 238 will not scale well if more security context options are added in the future.

Reply:
@ashb Actually I was wrong, this was causing integrations test to fails so I reverted to the previous behavior while taking into account the potential issue that you pointed out.